src/gui_common/charts/line/LineChart.cs

```diff
@@ -577,8 +577,11 @@ private void CreateDropDownLegend(string title)
     /// </summary>
     private void RenderChart()
     {
-        if (dataSets.Count <= 0)
+        if (dataSets == null || dataSets.Count <= 0)
```

Comment:
Instead of checking `dataSets.Count <= 0`, check if shown count > 0.

I'm able to produce this, where `Count = 3` but Shown ones =0
![image](https://user-images.githubusercontent.com/60734649/131221015-44c88b87-e492-4532-b6e2-d5f6ce6ecf49.png)

```suggestion
        if (dataSets?.All(d => d.Value.Draw != true) != false)
```

Reply:
That's a good idea, I only had the graph's available datasets in mind but not the _shown_ datas when implementing this.